src/cow-react/utils/number.ts

```diff
@@ -0,0 +1,7 @@
+import { DECIMAL_SEPARATOR } from '@cow/constants/format'
+
+export function decomposeDecimal(decimal: string): [string, string] {
+  const [integerPart, decimalPart] = decimal.split(DECIMAL_SEPARATOR)
+
+  return [integerPart, decimalPart]
+}
```

Comment:
As noted by Elena, this does not take into account cases where there are less decimals that what we want to display

I suggest to add here an option, or another fn, to pad/drop decimals based on a parameter

Reply:
I see you went for it and created another PR. Thanks!